app/src/main/java/org/wikipedia/dataclient/mwapi/UserContributions.kt

```diff
@@ -1,11 +1,12 @@
 package org.wikipedia.dataclient.mwapi
 
-import com.google.gson.annotations.SerializedName
-
 class UserContributions {
-    @SerializedName("user")
-    var user: String? = null
-
-    @SerializedName("title")
-    var title: String? = null
+    val user: String = ""
+        get() { return field.orEmpty() }
+    val title: String = ""
+        get() { return field.orEmpty() }
+    val timestamp: String = ""
+        get() { return field.orEmpty() }
+    val comment: String = ""
+        get() { return field.orEmpty() }
```

Comment:
It should be fine to remove the `get() { return field.orEmpty() }` since the `val` values cannot be reassigned and the default value is `empty`.

Reply:
Your statement is not exactly right: these values can definitely be reassigned, when they are deserialized from JSON.  And if the JSON response does not contain one of these fields, how sure are you that it will receive the default `empty` value?